src/starcheck.pl

```diff
@@ -678,10 +678,16 @@ sub json_obsids{
         my $att_ok = make_pcad_attitude_check_report(
             $backstop, $or_file, $mm_file, $char_file, $att_report);
         if ($att_ok){
-            $out .= "<A HREF=\"${att_report}\">[OK] Coordinates as expected.</A>\n\n";
+            $out .= "<A HREF=\"${att_report}\">[OK] Coordinates as expected.</A>\n";
         }
         else{
-            $out .= "<A HREF=\"${att_report}\">[NOT OK] Coordinate mismatch or error.</A>\n\n";
+            $out .= "<A HREF=\"${att_report}\">[NOT OK] Coordinate mismatch or error.</A>\n";
+        }
+        if (check_characteristics_date($char_file, $date[0])){
+            $out .= "<A HREF=\"${att_report}\">[OK] Characteristics file newer than 30 days</A>\n\n";
```

Comment:
Good catch, but technically the characteristics file is not available as a relative link since it would need to be placed in the starcheck output directory.